orc8r/cloud/deploy/mini-magma/hosts.yml

```diff
@@ -0,0 +1,7 @@
+---
+all:
+  hosts: 192.168.5.70
```

Comment:
This IP will change per-deployment. I would add a comment here that it needs to be updated (just in case people don't read the readme completely)

Reply:
Sure @mattymo, I will update it with a comment as per your suggestion.